autoload/vista/popup.vim

```diff
@@ -8,7 +8,10 @@ let s:popup_delay = get(g:, 'vista_floating_delay', 100)
 
 function! s:ClosePopup() abort
   if exists('s:popup_winid')
-    call popup_hide(s:popup_winid)
+    call popup_close(s:popup_winid)
+    try
+      unlet s:popup_winid
```

Comment:
This try-catch should be unnecssary.

I can merge this if it does work for you, other people might also benefit from it :(.

Reply:
Yeah you're right. I added it after hitting an error there at one stage, but that was while debugging and re-sourcing the script a lot, I suspect I just got the plugin into an invalid state, I can't reproduce it now.